packages/fe-fpm-writer/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "include": ["src"],
+    "include": ["src/**/*.ts"],
```

Comment:
Why this change?
We have `src` in every other project e.g. https://github.com/SAP/open-ux-tools/blob/main/packages/odata-service-writer/tsconfig.json#L2

Reply:
I tried to get rid of my local eslint problems, and had tried several things. When reading the [documentation](https://www.typescriptlang.org/tsconfig#include), I thought it better to restrict to the ts files here. But as .ts, .tsx, and .d.ts are the default we can also neglect the suffix. Indeed, it is not the only tsconfig file at open-ux-tools which uses the suffix. Finally, it might better to work without suffix as I can also find a glob definition under https://github.com/SAP/open-ux-tools/blob/main/tsconfig.json.

Reply:
@tobiasqueck all done